app/__init__.py

```diff
@@ -1,3 +1,58 @@
-from flask import Flask
+from flask import Flask, jsonify
 
 app = Flask(__name__)
+
+user = [
+    {
+        "name": "betty",
+        "email": "betty@email.com",
+        "password": "qwe123",
+        "confirm_password": "qwe123",
+        "request": [
+            {
+                "title": "Leaking pipe",
+                "type": "Repair",
+                "description": "Rusty pipe",
+                "category": "plumbing",
+                "area": "Block A"
+            }
+        ]
+    }
+]
+
+
+# Create user
+@app.route('/user', method=['POST'])
+def create(user):
+    pass
+
+
+# # Get user
+# @app.route('/user/<string:name>')
+# def get_user(user):
+#     return jsonify({"user": user})
+
+
+# Get all users
+@app.route('/user')
```

Comment:
From the rubric posted you are not supposed to have a route for getting all users
https://docs.google.com/document/d/1rZPo0ONvZco_ILlNs62UZNwrZCtNsyqU9IEvjKNlSp8/